xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceAccountService.java

```diff
@@ -201,8 +195,16 @@ public TradeHistoryParams createFundingHistoryParams() {
           (Long)
               exchange.getExchangeSpecification().getExchangeSpecificParametersItem("recvWindow");
 
+      Integer limit = null;
+
+      if (params instanceof TradeHistoryParamLimit) {
+        TradeHistoryParamLimit hpl = (TradeHistoryParamLimit) params;
+        limit = hpl.getLimit();
+      }
+
       boolean withdrawals = true;
       boolean deposits = true;
+      boolean airdrops = false;
```

Comment:
if `HistoryParamsFundingType#getType` is not set then this should return all funding history